apps/staking/src/components/stake/tables/PoolPerformanceTable.tsx

```diff
@@ -52,76 +53,136 @@ const PoolPerformanceTable: FC<PoolPerformanceTableProps> = ({
     const thRef = useRef<HTMLTableCellElement>();
     const tableRef = useRef<HTMLDivElement>();
     const threshold = useVisibilityThreshold(tableRef.current, thRef.current);
-    const borderColor = useColorModeValue('white', 'gray.700');
     const colSpans = 11;
+    const headerColor = useColorModeValue('header', 'dark.gray.primary');
+    const borderColor = useColorModeValue(
+        'transparent',
+        'dark.gray.quaternary'
+    );
+    const topBorderColor = useColorModeValue(
+        'transparent',
+        'dark.gray.quinary'
+    );
+    const buttonHoverColor = useColorModeValue('gray.90', 'dark.gray.quinary');
+
+    const thProps: TableColumnHeaderProps = {
+        borderColor: topBorderColor,
+        bg: headerColor,
+        textTransform: 'none',
+        fontSize: 'md',
+        fontWeight: 400,
+        fontFamily: theme.fonts.body,
+        paddingTop: 2,
+        paddingBottom: 2,
+    };
+
     return (
-        <TableResponsiveHolder ref={tableRef}>
+        <TableResponsiveHolder
+            ref={tableRef}
+            borderColor={borderColor}
+            borderWidth="1px"
+            borderRadius="3px"
```

Comment:
@nevendyulgerov Is this the correct `border-radius`? Because I think the style should be `6px`

<img width="1112" alt="Screenshot 2023-08-11 at 11 12 36" src="https://github.com/cartesi/explorer/assets/13269955/79ba4abd-4ba1-4742-ac58-784fedc18fef">

Reply:
Good catch @dandheedge , I'll adjust that in the next PR.